lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_get_store_app_sizes.rb

```diff
@@ -100,6 +100,10 @@ def self.output
           # Define the shared values you are going to provide
           
         end
+
+        def self.return_type
+          :hash
+        end
```

Comment:
_NB: this is not related to this PR, but since I saw the missing implementation here while I was creating the other new linting action I figured I could do a quick-fix where I was at it._